.github/workflows/script/solo_smoke_test.sh

```diff
@@ -0,0 +1,228 @@
+
+  if [ -s "private_key_without_quote_final.txt" ]; then
+    echo "File private_key_without_quote_final.txt exists and not empty"
+    return 0
+  else
+    echo "File private_key_without_quote_final.txt does not exist or empty"
+    return 1
+  fi
+}
+
+function clone_smart_contract_repo ()
+{
+  echo "Clone hedera-smart-contracts"
+  if [ -d "hedera-smart-contracts" ]; then
+    echo "Directory hedera-smart-contracts exists."
+  else
+    echo "Directory hedera-smart-contracts does not exist."
+    git clone https://github.com/hashgraph/hedera-smart-contracts --branch only-erc20-tests
+  fi
+}
+
+function clone_sdk_repo ()
+{
+  echo "Clone hedera-sdk-js"
+  if [ -d "hedera-sdk-js" ]; then
+    echo "Directory hedera-sdk-js exists."
+  else
+    echo "Directory hedera-sdk-js does not exist."
+    git clone https://github.com/hashgraph/hedera-sdk-js --branch v2.53.0
+    cd hedera-sdk-js
+    npm install --save @hashgraph/sdk
+    cd -
+  fi
+}
+
+# clone hedera local node repo and call function create_account_and_extract_key
+# to extract private keys
+function clone_local_node_repo ()
+{
+  echo "Clone hedera local node"
+  if [ -d "hedera-local-node" ]; then
+    echo "Directory hedera-local-node exists."
+  else
+    echo "Directory hedera-local-node does not exist."
+    git clone https://github.com/hashgraph/hedera-local-node --branch v2.32.0
+  fi
+  cd hedera-local-node
+  npm install
+
+  function_name=create_account_and_extract_key
+  retry_function 5
+  cd -
+}
+
+# retry the function saved in function_name for few times
+function retry_function ()
+{
+  local num=$1
+  for ((i=1; i<=$num; i++)); do
+    return_code=$( ( $function_name > retry.log; echo $? ))
+    # Use the return code
+    if [[ $return_code -eq 0 ]]; then
+      echo "Function $function_name is successful"
+      return
+    else
+      echo "Function $function_name failed with return code $return_code"
+    fi
+    echo "Retry $function_name in 2 seconds"
+    sleep 2
+  done
+  echo "Function $function_name failed after 5 retries"
+  exit 1
+}
+
+function setup_smart_contract_test ()
+{
+  echo "Setup smart contract test"
+  CONTRACT_TEST_KEYS=$(cat hedera-local-node/private_key_without_quote_final.txt)
+
+  cd hedera-smart-contracts
+
+  npm install
+  npx hardhat compile
+
+  echo "Remove previous .env file"
+  rm -f .env
+
+  echo "Build .env file"
+
+  echo "PRIVATE_KEYS=\"$CONTRACT_TEST_KEYS\"" > .env
+  echo "RETRY_DELAY=5000 # ms" >> .env
+  echo "MAX_RETRY=5" >> .env
+  cat .env
+  cd -
+}
+
+function background_keep_port_forward ()
+{
+  for i in {1..40}; do
+    echo "Enable port forward round $i"
+    enable_port_forward
+    sleep 2
+    ps -ef |grep port-forward
+  done &
+}
+
+function start_background_transactions ()
+{
+  echo "Start background transaction"
+  # generate accounts every 3 seconds as background traffic for two minutes
+  # so record stream files can be kept pushing to mirror node
+  cd hedera-local-node
```

Comment:
just use Solo to generate the accounts, we have a `solo account create` function.  can just call it multiple times.   Or, even better would be to enable the mirror node monitor, my understanding is that it wil automatically push transactions.